.yamato/build-macos-all.yml

```diff
@@ -0,0 +1,14 @@
+name: Build protobuf - macOS 10.12 x64 - macOS 11.0 arm64
+
+agent:
+  type: Unity::VM::osx
+  image: platform-foundation/mac-bokken:latest
+  flavor: b1.medium
+
+artifacts:
+  builds_mac:
+    paths:
+      - builds/**
+
+commands:
+- mono bee/bee.exe
```

Comment:
This works for now, since you only ever build for host platform. Generally though, you'd tell bee explicitly what it should build, as not specifying anything will build every single thing in the build graph.

Reply:
The intent was to build anything on the build graph that can be built on that platform :)

Reply:
Generally you don't want that because you'd build duplicates, as many platforms support cross compiling from both Windows and Mac. It's ok for this lib since you're only building for the editor platforms but this might change in the future when we add support for bee to build Linux binaries on Windows.